library/src/main/java/com/rengwuxian/materialedittext/MaterialEditText.java

```diff
@@ -331,6 +331,14 @@ public MaterialEditText(Context context, AttributeSet attrs, int style) {
     init(context, attrs);
   }
 
+  public String getString() {
+    return getText().toString().trim();
```

Comment:
Yeah but when you deal with user profile forms, you need to trim the strings. Either by extending MET, or creating a static function.